tests/test_repodict.py

```diff
@@ -37,13 +37,9 @@ def setUp(self):
         self.full_set = {self.x, self.xx, self.y, self.z}
 
     def test_any_enabled(self):
-        self.assertTrue(self.repos.any_enabled())
+        self.assertTrue(self.repos._any_enabled())
         self.repos.get_matching("*").disable()
-        self.assertFalse(self.repos.any_enabled())
-
-    def test_enabled(self):
```

Comment:
Because self.repos.enabled() as redundant def of iter_enabled() (API) and therefore it was removed. Both generate output by identical way, but one return list and second one tuple. The test only does that change tuple to list and compare output.